presto-orc/src/test/java/com/facebook/presto/orc/OrcTester.java

```diff
@@ -578,6 +578,11 @@ private static void assertFileContentsPresto(
 
                 assertFalse(iterator.hasNext());
             }
+            catch (UnsupportedOperationException e) {
+                // Disable the tests on the StreamReaders that do not support filter push down yet.
+                assertTrue(e.getMessage().startsWith("setFilterAndChannel is not supported"));
```

Comment:
A cleaner way is to override the failing tests methods in `TestAriaOrcReader` to have empty bodies.